tests/tizen_capi/unittest_tizen_capi.cc

```diff
@@ -4526,55 +4673,58 @@ TEST (nnstreamer_capi_singleshot, invoke_dynamic_success_02_p)
   status = ml_tensors_data_create (in_info, &input);
   EXPECT_EQ (status, ML_ERROR_NONE);
 
-  float tmp_input2[] = { 1.0, 2.0, 3.0, 4.0, 5.0 };
-  float *output_buf2;
-  status = ml_tensors_data_set_tensor_data (input, 0, tmp_input2,
-      5 * sizeof (float));
+  {
+    float tmp_input2[] = { 1.0, 2.0, 3.0, 4.0, 5.0 };
+    float *output_buf2;
+    status = ml_tensors_data_set_tensor_data (input, 0, tmp_input2,
+        5 * sizeof (float));
 
-  ml_tensors_info_get_count (in_info, &tmp_count);
-  ml_tensors_info_get_tensor_type (in_info, 0, &tmp_type);
-  ml_tensors_info_get_tensor_dimension (in_info, 0, tmp_dim);
+    ml_tensors_info_get_count (in_info, &tmp_count);
+    ml_tensors_info_get_tensor_type (in_info, 0, &tmp_type);
+    ml_tensors_info_get_tensor_dimension (in_info, 0, tmp_dim);
 
-  EXPECT_EQ (tmp_count, 1U);
-  EXPECT_EQ (tmp_type, ML_TENSOR_TYPE_FLOAT32);
-  EXPECT_EQ (tmp_dim[0], 5U);
-  EXPECT_EQ (tmp_dim[1], 1U);
-  EXPECT_EQ (tmp_dim[2], 1U);
-  EXPECT_EQ (tmp_dim[3], 1U);
+    EXPECT_EQ (tmp_count, 1U);
+    EXPECT_EQ (tmp_type, ML_TENSOR_TYPE_FLOAT32);
+    EXPECT_EQ (tmp_dim[0], 5U);
+    EXPECT_EQ (tmp_dim[1], 1U);
+    EXPECT_EQ (tmp_dim[2], 1U);
+    EXPECT_EQ (tmp_dim[3], 1U);
 
-  status =
-      ml_single_invoke_dynamic (single, input, in_info, &output, &out_info);
-  EXPECT_EQ (status, ML_ERROR_NONE);
+    status =
+        ml_single_invoke_dynamic (single, input, in_info, &output, &out_info);
+    EXPECT_EQ (status, ML_ERROR_NONE);
 
-  ml_tensors_data_get_tensor_data (output, 0, (void **) &output_buf2,
-      &data_size);
+    ml_tensors_data_get_tensor_data (output, 0, (void **) &output_buf2,
+        &data_size);
 
-  EXPECT_EQ (output_buf2[0], 3.0f);
-  EXPECT_EQ (output_buf2[1], 4.0f);
-  EXPECT_EQ (output_buf2[2], 5.0f);
-  EXPECT_EQ (output_buf2[3], 6.0f);
-  EXPECT_EQ (output_buf2[4], 7.0f);
-  EXPECT_EQ (data_size, 5 * sizeof (float));
+    EXPECT_EQ (output_buf2[0], 3.0f);
+    EXPECT_EQ (output_buf2[1], 4.0f);
+    EXPECT_EQ (output_buf2[2], 5.0f);
+    EXPECT_EQ (output_buf2[3], 6.0f);
+    EXPECT_EQ (output_buf2[4], 7.0f);
+    EXPECT_EQ (data_size, 5 * sizeof (float));
```

Comment:
For floating point values, EXPECT_NEAR/ASSERT_NEAR is more appropriate than EXPECT_EQ/ASSERT_EQ.

Reply:
@myungjoo I submit another commit for resolving yout comment. Thanks.